src/parse/rules/python_rules.build_defs

```diff
@@ -181,7 +181,8 @@ def python_test(name, srcs, data=None, resources=None, deps=None, labels=None, s
     """Generates a Python test target.
 
     This works very similarly to python_binary; it is also a single .pex file
-    which is run to execute the tests. The tests are run via unittest.
+    which is run to execute the tests. The tests are run via either unittest or pytest, depending
+    on which is set for the python.testrunner property in .plzconfig.
```

Comment:
nit: imo delete  `in .plzconfig.` because you can override it for a given folder which is likely to be a common use case as one starts writing new tests in pytest while still using unittest ones elsewhere or vice-versa

Reply:
I've expanded it a bit to better describe available options - I wanted to explain a bit how you'd change it. Agreed that it's useful to describe the per-folder case although I suspect a lot of potential users will be using one or the other already.